src/modules/job-info/job_state.c

```diff
@@ -1745,45 +1677,126 @@ static int job_events_annotations (struct job_state_ctx *jsctx,
     return 0;
 }
 
-static void job_events_continuation (flux_future_t *f, void *arg)
+static int job_events_process (struct job_state_ctx *jsctx, json_t *events)
```

Comment:
The `job_events_process()` return code is never checked so should it be void?

Reply:
the return code is checked in `job_events_continuation()`

Reply:
sorry - missed that